### DISCUSSION TITLE
Possible issue?

### DISCUSSION THREAD

**Discussion by ddman2** [LeoKlaus/plappa#127] [General]
I'm on the beta on iOS 17.5.1.  Changes to accent color or theme don't seem to do anything.

**Comment by LeoKlaus** [comment]
Thank you for reporting this. There are a few weird bugs on 17.5.1 including this. Others have reported that restarting the app applied the accent color.

As this issue is not present in other versions of iOS (including iOS 18), I'll consider this an iOS bug for now.
